backup-script.js

```diff
@@ -0,0 +1,289 @@
+// ==================================================
+// Settings & Constants
+// API keys, URLs, and initial setup variables
+// ==================================================
+document.addEventListener("DOMContentLoaded", () => {
+
+const myHeaders = new Headers();
+myHeaders.append('x-api-key', '0c625ab60e6a40a1a4b6cc2e4a5fe9b0');
+// Tells the API that we want the data in JSON format
+myHeaders.append('Content-Type', 'application/json')
+
+const requestOptions = {
+    method: 'GET', // We want to GET (fetch) data
+    redirect: 'follow',
+    headers: myHeaders // API-key sends as a HTTP-header, more safe without the key in the URL
+};
```

Comment:
great that you have investigated in this. We will learn more about storing API keys in a secret and secure way